adi/ad4630.py

```diff
@@ -80,17 +79,27 @@ def __init__(self, uri="", device_name="ad4630-24"):
             self._rxadc = self._ctx.find_device(device_name)
 
         _channels = []
-        self.output_bits = []
+        self.output_bits = [ch.data_format.bits for ch in self._ctrl.channels]
+        self._has_common = False
+        self._is_dual = None
         for ch in self._ctrl.channels:
-            self.output_bits.append(ch.data_format.bits)
             self._rx_channel_names.append(ch.id)
+            if "common" in ch.name:
+                self._has_common = True
             if "differential" in ch.name:
                 _channels.append((ch.id, self._diff_channel(self._ctrl, ch.id)))
                 if "0" in ch.name:
                     self.chan0 = self._diff_channel(self._ctrl, ch.name)
                 if "1" in ch.name:
                     self.chan1 = self._diff_channel(self._ctrl, ch.name)
 
+        if (len(self._ctrl.channels) == 4) or (
+            len(self._ctrl.channels) == 2 and not self._has_common
+        ):
+            self._is_dual = True
+        else:
+            self._is_dual = False
+

```

Comment:
Can you change this if/else to a one liner:
```python
self._is_dual = (len(self._ctrl.channels) == 4) or (len(self._ctrl.channels) == 2 and not self._has_common)
```

Reply:
Done.

Reply:
This isn't a formatting issue. You can collapse the if/else to a single line and directly assign ```self._is_dual```

Reply:
Got it. Done.